RiotSharpTest/RiotApiTest.cs

```diff
@@ -176,37 +177,67 @@ public void GetChampionAsync_Test()
             });
         }
 
-        [Ignore] //Depricated and being refactored in #411
         [TestMethod]
         [TestCategory("RiotApi")]
-        public void GetMasteryPages_Test()
+        public void GetMasteryPages_ExistingSummonerId_HasMasteryPages()
         {
             EnsureCredibility(() =>
             {
-                var masteries = api.GetMasteryPages(RiotApiTestBase.summonersRegion, RiotApiTestBase.summonerIds);
+                var pages = api.GetMasteryPages(RiotApiTestBase.summonersRegion, RiotApiTestBase.summoner1Id);
 
-                Assert.IsNotNull(masteries);
-                Assert.AreEqual(RiotApiTestBase.summonerIds.Distinct().Count(), 
-                    masteries.Count);
+                Assert.IsNotNull(pages);
+                Assert.IsTrue(pages.Count >= 1 && pages.Count <= 20);
             });
         }
 
-        [Ignore] //Depricated and being refactored in #411
+        [TestMethod]
+        [TestCategory("RiotApi")]
+        public void GetMasteryPages_InvalidSummonerId_ThrowsResouceNotFound()
+        {
```

Comment:
Could you implement `EnsureCredibility()` here. I know it won't be need here probably, but the server could still return 500 or 503.